### ISSUE TITLE
heibaizhibo.com

### ISSUE THREAD

**Issue by TaoziDB** [ytdl-org/youtube-dl#29162]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2021.05.16**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single live stream: https://www.heibaizhibo.com/anchorLive/513780
- Single live stream: https://www.heibaizhibo.com/anchorLive/543655
- Live streams list: https://www.heibaizhibo.com/anchorLive

## Description

https://www.heibaizhibo.com/anchorLive

sports live streams site from China

**Comment by SuperSonicHub1** [comment]
@TaoziDB I'm getting 403s and the stream never finishes loading. I live in the US; is the site geoblocked?

**Comment by TaoziDB** [comment]
@SuperSonicHub1 Yes, the site is blocking US IP address, you need China IP address.
Japan, Singapore IP address and Taiwan IP address work too, which are easier to get than China IP address. 
nowadays, most VPN don't have China IP address,

**Comment by SuperSonicHub1** [comment]
@TaoziDB
> Yes, the site is blocking US IP address, you need China IP address.

So I can't help you, as I don't have the cash for a VPN. If you make another extractor request please leave information about geoblocking in the description. Any Asian devs willing to help?
